Automattic/jetpack
# Add newsletter access column to post  edit page

Fixes https://github.com/Automattic/wp-calypso/issues/75788
Relates to https://github.com/Automattic/wp-calypso/pull/76382

## Proposed changes:

Adds a "Newsletter Access" column to the post edit page.

![romarioraffearn wordpress com_wp-admin_edit php](https://user-images.githubusercontent.com/7076981/232914326-27a61ccc-fbea-4bf7-9551-c3a177bced5c.png)


### Other information:


## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Add the patch to your sandbox
* On a site with newsletter enabled, verify that the "Newsletter access" column is displayed in the post edit page
* Update the access level for a post 
* Refresh the post edit page then verify the access level for the post is updated correctly
* Disable newsletter ( you can turn off Autoproxxy ) and verify that the "Newsletter access" column is **not** displayed on the post edit page
